mava/components/component_groups.py

```diff
@@ -6,7 +6,7 @@
     executing.RecurrentExecutorObserve,
 ]
 
-# The components that a needed for IPPO with the Huber value loss.
+# The components that a needed for the Huber value loss.
 huber_value_loss_components = [
```

Comment:
Spoke to @DriesSmit about this offline: that we should remove single components from the groups file. This is just a reminder to do that before we merge.

Reply:
Ok this is done :)